### ISSUE TITLE
Flex subviews aren't the same height when superview doesn't have height

### ISSUE THREAD

**Issue by xueduany** [facebook/react-native#6229]
I try it on React-Native 0.16, like
<Div className="good">
					<Div className="title" style={{backgroundColor: 'black'}}>宜</Div>
					<Div className="content">
					kqwjelqkwjelkqjw
					kqwjelqkwjelkqjw
					kqwjelqkwjelkqjw
					kqwjelqkwjelkqjw
					kqwjelqkwjelkqjw
					kqwjelqkwjelkqjw
					kqwjelqkwjelkqjw
					kqwjelqkwjelkqjw
					kqwjelqkwjelkqjw
					kqwjelqkwjelkqjw
					kqwjelqkwjelkqjw
					kqwjelqkwjelkqjw
					kqwjelqkwjelkqjw
					kqwjelqkwjelkqjw
					kqwjelqkwjelkqjw
					kqwjelqkwjelkqjw
					kqwjelqkwjelkqjw
					</Div>
				</Div>

.good, .bad {
	display: flex;
	flex-direction: row;
}

.good .title, .bad .title {
	flex: 0.5;
	font-weight: bold;
	font-size: 2rem;
	display: flex;
}

.good .content, .bad .content {
	flex: 1;
	padding-right: 10px;
	padding-top: 1px;
	font-size:15pt;
}

but two flex item not same height

**Comment by grabbou** [comment]
I think he might be using `react-native-web` kind of thing, but I have no clue on using pure CSS (looks neat!)

**Comment by grabbou** [comment]
Also - small tip - try upgrading to latest React to see if the error is still present :)

**Comment by xueduany** [comment]
No, you mistake my means, first I build a project can translate react style H5 code into react-native real code. So the code above you see is just what I mean.
The real react-native code is below

<View style={{display:'flex',flexDirection:'row'}}>
				  	<View style={{flex: 1,backgroundColor: 'red', alignItems: 'stretch'}}><Text>qwjel;qjw</Text></View>
				  	<View style={{flex: 1,backgroundColor: 'green'}}>
				  		<View><Text>kqwjelkqjwelkjqwlke</Text></View>
				  		<View><Text>kqwjelkqjwelkjqwlke</Text></View>
				  		<View><Text>kqwjelkqjwelkjqwlke</Text></View>
				  		<View><Text>kqwjelkqjwelkjqwlke</Text></View>
				  		<View><Text>kqwjelkqjwelkjqwlke</Text></View>
				  		<View><Text>kqwjelkqjwelkjqwlke</Text></View>
				  		<View><Text>kqwjelkqjwelkjqwlke</Text></View>
				  		<View><Text>kqwjelkqjwelkjqwlke</Text></View>
				  		<View><Text>kqwjelkqjwelkjqwlke</Text></View>
				  		<View><Text>kqwjelkqjwelkjqwlke</Text></View>
				  		<View><Text>kqwjelkqjwelkjqwlke</Text></View>
				  		<View><Text>kqwjelkqjwelkjqwlke</Text></View>
				  		<View><Text>kqwjelkqjwelkjqwlke</Text></View>
				  	</View>
				  </View>

but the flex style is not working as I expect that
